examples/webgl_materials_physical_transmission.html

```diff
@@ -170,6 +174,22 @@
 
 					} );
 
+				gui.add( params, 'shininess', 0, 5, 0.01 )
```

Comment:
This parameter is in the [ 0, 1 ] range only.

It remains to discuss whether it should be a property of the material, or only added as-needed in an extension.

Reply:
Thanks, I overlooked that the glTF extension also defines that it's in 0 to 1. Updated.